ex/high_throughput_inference/standalone_workermanager.py

```diff
@@ -75,22 +179,65 @@
     to_worker_fli_serialized = to_worker_fli.serialize()
     ddict["to_worker_fli"] = to_worker_fli_serialized
 
-    torch_worker = cloudpickle.loads(base64.b64decode(args.worker_class.encode('ascii')))()
+    arg_worker_type = cloudpickle.loads(
+        base64.b64decode(args.worker_class.encode("ascii"))
+    )
 
     dfs = DragonFeatureStore(ddict)
     comm_channel = DragonFLIChannel(to_worker_fli_serialized)
 
     os.environ["SSFeatureStore"] = base64.b64encode(pickle.dumps(dfs)).decode("utf-8")
     os.environ["SSQueue"] = base64.b64encode(to_worker_fli_serialized).decode("utf-8")
 
-    config_loader = EnvironmentConfigLoader()
+    ss_config_loader = EnvironmentConfigLoader()
 
-    worker_manager = WorkerManager(
-        config_loader=config_loader,
-        worker=torch_worker,
-        as_service=True,
-        cooldown=10,
+    dispatcher = create_request_dispatcher(
+        batch_size=args.batch_size,
+        batch_timeout=args.batch_timeout,
         comm_channel_type=DragonCommChannel,
-        device = args.device,
+        worker_type=arg_worker_type,
+        config_loader=ss_config_loader,
     )
-    worker_manager.execute()
+
+    wms = []
+    worker_device = args.device
+    for wm_idx in range(args.num_workers):
+        # if args.num_workers > 0:
+        #     worker_device = f"{args.device}:{wm_idx}"
+        worker_manager = create_worker_manager(
+            worker_type=arg_worker_type,
+            config_loader=ss_config_loader,
+            device=worker_device,
+            dispatcher_queue=dispatcher.task_queue,
+        )
+        wms.append(worker_manager)
+
+    wm_affinity: list[int] = []
+    disp_affinity: list[int] = []
+
+    # This is hardcoded for a specific type of node!
```

Comment:
Could we just expand this comment to explain something like `we're configuring based on NUMA characteristics for machine Xyz`